examples/generative/stylegan.py

```diff
@@ -58,7 +69,11 @@ def log2(x):
 train_step_ratio = {k: batch_sizes[2] / v for k, v in batch_sizes.items()}
 
 
-ds_train = tfds.load("celeb_a", split="train")
+# ds_train = tfds.load("celeb_a", split="train")
+# As of Dec-3 2021, the [TFDS Celeb A](https://www.tensorflow.org/datasets/catalog/celeb_a) 
+# dataset gives an `wrong checksum` error, use this library as a work around
+ds_train = tfds.load('celeba_backup', split='train')
```

Comment:
My experience with TFDS is that it almost never works as intended. I recommend instead not using it. In this example I wrote we grab it from gdrive: https://keras.io/examples/generative/dcgan_overriding_train_step/ I suggest doing that here too.

Reply:
And I was just learning to love TFDS.... but updated the code to use GDrive directly